### PR TITLE
update copy and confirmation page logic

### PR DESCRIPTION
**Pull request by rianfowler**
## Not signed in or not verified 
2018-02-05-10-05-localhost_3001

## signed in and verified
2018-02-05-09-52-localhost_3001

### CODE REVIEW

File: src/js/vic-v2/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -33,6 +34,7 @@ class ConfirmationPage extends React.Component {
   }
 
   render() {
+    const signedIn = _get(this.props.saveInProgress, 'user.login.currentlyLoggedIn', false);
```
Comment (rianfowler) [inline src/js/vic-v2/containers/ConfirmationPage.jsx]: Not sure how to get this value so I copied what's on the introduction page. LMK if this has to be done another way @jbalboni

Comment (jbalboni) [reply src/js/vic-v2/containers/ConfirmationPage.jsx]: I think you'll need to add `user` to the `mapStateToProps` function below (it's at `state.user`) and get it from there.

File: src/js/vic-v2/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import _get from 'lodash/get';
```
Comment (jbalboni) [inline src/js/vic-v2/containers/ConfirmationPage.jsx]: We normally `import _ from 'lodash/fp';`. We use the webpack babel plugin that transforms those imports into ones like this that just import the specific functions being used.

**Review by jbalboni — APPROVED** [review]
Couple things, but LGTM.

File: src/js/vic-v2/containers/ConfirmationPage.jsx [hunk 3]
```diff
@@ -45,22 +47,29 @@ class ConfirmationPage extends React.Component {
...
+            In the meantime, you can print a temporary digital Veteran ID Card.</p>
+          <VeteranIDCard/>
+          <button type="button" className="va-button-link" onClick={() => window.print()}>Print your temporary Veteran ID Card.</button>
+        </div>}
+        {(!verified || !signedIn) && <div>
+          <p>We’ll review your application to verify your eligibility. If you’re eligible for a Veteran ID Card, you should receive your card by mail in about 60 days.<br/>
+            We’ll send you emails updating you on the status of your application. You can also print this page for your records.</p>
           <p>To be eligible for a Veteran ID Card, you must have separated under honorable conditions. If you have an other than honorable discharge, you can apply for an upgrade or correction.<br/>
             <a href="/discharge-upgrade-instructions" target="_blank">Find out how to apply for a discharge upgrade</a>.</p>
           <p>If you uploaded a copy of your discharge document, we'll use it to verify that you were honorably discharged.</p>
```
Comment (cehsu) [inline src/js/vic-v2/containers/ConfirmationPage.jsx]: Please replace the apostrophe in "we'll" with a curly apostrophe.

**Pull request merged**